apps/google-analytics-4/frontend/src/components/main-app/analytics-app/AnalyticsApp.styles.ts

```diff
@@ -9,28 +9,26 @@ const load = keyframes`
 	}
 `;
 
+const loaderWidth = 50;
 export const styles = {
   loader: css({
     height: '100%',
     width: '100%',
     ':before': {
       content: "''",
       position: 'absolute',
-      top: 0,
-      right: 'auto',
-      bottom: 0,
-      left: 'auto',
+      top: `calc(50% - ${loaderWidth / 2}px)`,
+      left: `calc(50% - ${loaderWidth / 2}px)`,
       margin: 'auto',
-      height: '50px',
-      width: '50px',
-      borderWidth: '5px',
+      height: `${loaderWidth}px`,
+      width: `${loaderWidth}px`,
+      borderWidth: `${loaderWidth / 10}px`,
```

Comment:
went with 10% relative sizing here to support more contexts: 

https://user-images.githubusercontent.com/492573/225668713-21ae57be-ab48-4ec4-9539-36bd943538f5.mov